lib/sinatra/show_exceptions.rb

```diff
@@ -24,23 +24,22 @@ def call(env)
 
       if prefers_plain_text?(env)
         content_type = "text/plain"
-        exception_string = dump_exception(e)
+        exception = dump_exception(e)
       else
         content_type = "text/html"
-        exception_string = pretty(env, e)
+        exception = pretty(env, e)
       end
 
       env["rack.errors"] = errors
-
-      # Post 893a2c50 in rack/rack, the #pretty method above, implemented in
-      # Rack::ShowExceptions, returns a String instead of an array.
```

Comment:
I think it's important to leave the comment, because it notes a surprising change. For people who are familiar with Sinatra, this could very easily look like a bug. I know that it's possible to find the reason by using `git blame`, but that won't always be easy if someone changes the line of code at some point.